src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+
+### Connected
+Page weight can also be effected by external services that are connected to the web page. Some of those services include CDN's, analytics, chat bots, forms, and other data collection and processing methods. I find this to be one of the fastest growing problem areas that result in bloated page weight. Many of these third party services use outdated, poorly written JavaScript and querying techniques that take much longer to execute than they should, and the site owner has little control over how that third party impacts the loading of a page. Suffice it to say that inquiring about how a service will affect your page loading performance is very important. So is testing their impact.
+
+### Caching
+[Caching](https://almanac.httparchive.org/en/2021/caching), are special services/processes that create copies of previously requested optimized assets and store them on remote server or on the client's browser/disk until needed again. Caching of optimized assets dramatically lowers page weight and page loading time because the asset is immediately available, removing the need to execute and entire request process.
+
+## Analysis
+
+As we post and parse statistical results, the data is often based on transfer sizes. However, we are employing decompressed sizes in this analysis whenever possible. Please note that many of the results listed below are similar or the same as those queries used in 2020.
+
+### Page weight by the numbers
+
+Looking at the page weight on both desktop and mobile devices, the deltas are due to the difference in the amount of transferred resources on the mobile device, the difference is generally small between them.
+
+{{ figure_markup(
+  image="bytes-distribution.png",
+  caption="Distribution of total bytes per page.",
+  description="Bar chart showing the distribution of the total bytes per page. Desktop pages tend to have more bytes throughout the distribution. The 10, 25, 50, 75, and 90th percentiles for mobile pages are: 409, 928, 1,923, 3,749, and 6,890 KB per page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=8302830&format=interactive",
+  sheets_gid="895329225",
+  sql_file="bytes_per_type_2021.sql"
+) }}
+
+The key data here shows we are closing in on 6.9 MB of page weight on mobile and 8.1 MB on desktop at the 90th percentile. The data is following an old and consistent trend: growth in page weight is on the upward trajectory that continues from the previous year.
```

Comment:
I would first state the median page weight and compare that to the previous year. You can also compare the 90th percentile to the previous year, but the 90th percentile this year alone does not tell you anything about a trend. If you have a number, maybe state it or add another chart showing the year-over-year change just for mobile devices (for each percentile).